docs/src/guides/drupal9/redis.md

```diff
@@ -1,149 +1,172 @@
+      $settings['redis.connection']['port'] = $redis['port'];
+
+      // Apply changes to the container configuration to make better use of Redis.
+      // This includes using Redis for the lock and flood control systems, as well
+      // as the cache tag checksum. Alternatively, copy the contents of that file
+      // to your project-specific services.yml file, modify as appropriate, and
+      // remove this line.
+      $settings['container_yamls'][] = 'modules/contrib/redis/example.services.yml';
+
+      // Allow the services to work before the Redis module itself is enabled.
+      $settings['container_yamls'][] = 'modules/contrib/redis/redis.services.yml';
+
+      // Manually add the classloader path, this is required for the container
+      // cache bin definition below.
+      $class_loader->addPsr4('Drupal\\redis\\', 'modules/contrib/redis/src');
+
+      // Use redis for container cache.
+      // The container cache is used to load the container definition itself, and
+      // thus any configuration stored in the container itself isn't available
+      // yet. These lines force the container cache to use Redis rather than the
+      // default SQL cache.
+      $settings['bootstrap_container_definition'] = [
+        'parameters' => [],
+        'services' => [
+          'redis.factory' => [
+            'class' => 'Drupal\redis\ClientFactory',
+          ],
+          'cache.backend.redis' => [
+            'class' => 'Drupal\redis\Cache\CacheBackendFactory',
+            'arguments' => ['@redis.factory', '@cache_tags_provider.container', '@serialization.phpserialize'],
+          ],
+          'cache.container' => [
+            'class' => '\Drupal\redis\Cache\PhpRedis',
+            'factory' => ['@cache.backend.redis', 'get'],
+            'arguments' => ['container'],
+          ],
+          'cache_tags_provider.container' => [
+            'class' => 'Drupal\redis\Cache\RedisCacheTagsChecksum',
+            'arguments' => ['@redis.factory'],
+          ],
+          'serialization.phpserialize' => [
+            'class' => 'Drupal\Component\Serialization\PhpSerialize',
+          ],
+        ],
+      ];
+    }
+   ```
+
+   You can customize your configuration further 
+   using the inline comments from the above example configuration.
+   For more information on possible configuration options, 
+   see the `README.txt` file delivered with the Redis module 
+   or the [official Redis documentation](https://redis.io/docs/). 
+
+## Verify Redis is running
+
+To verify that Redis is running:
+
+1. [Connect to your app with SSH](../../development/ssh/). 
+
+2. In your SSH terminal, run the following command:
+
+   ```bash
+   echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
+   ```
+   In the output, retrieve the value of the `host` property for your Redis relationship.
+
+3. Run the following command:
+
+   ```bash
+   redis-cli -h {{< variable "HOST" >}} info
```

Comment:
If the above is changed to start with `platform ssh`, this should be too. Not necessary otherwise.
```suggestion
   platform ssh -- redis-cli -h {{< variable "HOST" >}} info
```